### DISCUSSION TITLE
change the color of properties in WB editor

### DISCUSSION THREAD

**Discussion by mrvicam** [eclipse/windowbuilder#714] [Q&A]
How to change color theme in desing view of WB for properties of element? 
I change color theme of Eclipse.in DevStyle Theme.

**Comment by ptziegler** [comment]
Can you elaborate a little? Do you want to change the theme of a design element or of the editor itself?

> I change color theme of Eclipse.in DevStyle Theme.

What do you mean by "DevStyle Theme"? Is that a custom theme you're using?
